view/frontend/web/js/action/express-pay/update-quote-shipping-method-action.js

```diff
@@ -20,23 +20,26 @@ define(
          * @return void
          */
         return async function (shippingMethod = null) {
-            let newMethod = null;
+            let newMethod;
+            let ms = 100;
```

Comment:
Nitpick: Perhaps this should be called `timeout` for clarity?

Reply:
yeah this can be `timeout` or` `timeoutMS`

Reply:
I'm partial to `timeoutMS` or `timeoutMillis`

Reply:
updated to use `timeoutMS`